### ISSUE TITLE
Combination list: All X in the DB AND/OR All Y in the DB

### ISSUE THREAD

**Issue by willgearty** [learning-unlimited/ESP-Website#1796]
When using the comm panel (and perhaps the select user list as well), if you want to make a combination list, you can select "All X in the DB" but then all of your secondary options seem limited to the current program. Would it be possible to implement a way to select All X or Y in the DB?
For example, we want to be able to email all teachers AND volunteers in the database.

**Comment by willgearty** [comment]
Another example: we'd like to e able to email ALL students and ALL guardians (not just the guardians of students with accounts).

**Comment by willgearty** [comment]
I think this is a duplicate of #1654.

**Comment by benjaminjkraft** [comment]
Nope, this is different -- in this case you can't make the desired query via the interface; in #1654 you can make the query (a different one) but it errors.

**Comment by willgearty** [comment]
Oh, good call, must have misread my own issue...

**Comment by willgearty** [comment]
Unassigning due to inactivity.
